src/javascript/Validation/validation.js

```diff
@@ -55,6 +55,20 @@ const dateFieldValidation = (values, field) => {
     return respectEachConstraint ? undefined : error;
 };
 
+const emailFieldValidation = (values, field) => {
+    const error = 'invalidEmail';
+
+    const fieldValues = Array.isArray(values[field.name]) ? values[field.name] : [values[field.name]];
```

Comment:
can be moved inside the next if for optimization.
if we dont have constraint it's not necessary to transform value into array.

Also can we not detect if field.multiple ? instead of Array.isArray(values[field.name]) ?